### DISCUSSION TITLE
Failed to load resource map..

### DISCUSSION THREAD

**Discussion by Semiglas** [tailpress/tailpress#193] [Q&A]
Hi! I wanted to ask if you know why could i get this  error 

_**DevTools failed to load source map: Could not load content for https://webwhimstudio.com/wp-content/plugins/tailpress/js/cdn.global.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE**_

i don't even have that file in my DB , the only files are: 
![Снимок экрана 2023-06-16 в 21 29 20](https://github.com/jeffreyvr/tailpress/assets/87945783/8d63d05c-2550-40c1-8830-9d42ce395006)

Love tailwind immensely and very grateful for your work but this error somehow breaks my site in a very unexpected way all the time

**Comment by jeffreyvr** [comment]
Hi, I understand the confusion but the plugin your using is not affiliated with this project.
